frontend/EmpUI/src/features/users/user-table.tsx

```
@@ -1,27 +1,66 @@
+      const searchTerm = value.toLowerCase();
+      const filtered = data.filter((employee) => {
+        const roleMatch =
+          (searchTerm.includes("admin") && employee.role === 0) ||
+          (searchTerm.includes("employee") && employee.role === 2) ||
+          // Also match if user directly searches for "0" or "2"
+          (employee.role !== undefined &&
+            employee.role.toString() === searchTerm);
+
+        return (
+          // Name field
+          (employee.name && employee.name.toLowerCase().includes(searchTerm)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)